lib/modules/manager/bundler/artifacts.ts

```diff
@@ -32,14 +30,16 @@ import {
 const hostConfigVariablePrefix = 'BUNDLE_';
 
 function buildBundleHostVariable(hostRule: HostRule): Record<string, string> {
-  if (!hostRule.resolvedHost || hostRule.resolvedHost.includes('-')) {
+  if (!hostRule.resolvedHost) {
     return {};
```

Comment:
@rarkins , this is the line failing the test coverage. I wonder if checking the  "hostRule.resolvedHost" is even necessary. If not, I can remove the if statement. Otherwise, I'd appreciate any help creating a proper test.